.github/workflows/publish-c-apidocs.yml

```diff
@@ -21,27 +27,50 @@ jobs:
           sudo apt-get install libclang-cpp14
           wget https://www.doxygen.nl/files/doxygen-1.9.6.linux.bin.tar.gz
           tar xvzf doxygen-1.9.6.linux.bin.tar.gz
-      - name: Set commit ID
-        id: vars
-        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
       - name: Run doxygen
         run: |
           mkdir -p build/doxygen
           cd docs/c_cxx
           ../../doxygen-1.9.6/bin/doxygen
-      - uses: actions/checkout@v2
+      - name: Log source commit
+        run: git rev-parse --short HEAD > build/doxygen/html/source-version.txt
+      - name: Download deployed site
+        uses: dawidd6/action-download-artifact@v2
```

Comment:
why this instead of https://github.com/actions/download-artifact?

Reply:
I tried that first but AFAIK it only allows artifacts generated in the same workflow.

Reply:
DNABKKOFFICJ